platform/security/certificate/security-certificate-generator/src/main/java/org/codice/ddf/security/certificate/generator/CertificateCommand.java

```diff
@@ -160,14 +166,23 @@ public static String configureDemoCertWithDN(String[] dn) {
   public static String configureDemoCertWithDN(String[] dn, @Nullable String[] subjectAltNames) {
     CertificateSigningRequest csr = new CertificateSigningRequest();
     csr.setDistinguishedName(dn);
+
+    // Required for Chrome support (DDF-3104)
+    String commonName =
```

Comment:
✏️   As previously stated, I am worried that we are duplicating logic that is in PkiTools and bouncy castle.

How about ```String commonName = csr.getSubjectName().getRDNs(BCStyle.CN)[0].getFirst().getValue().toString() ```?

Might want to validate some of those steps but that should get you the common name portion without having to try to parse it yourself?